Description: A bug that treats tar warnings as errors has been fixed.
Author: Alexey Salmin <alexey.salmin@gmail.com>
Bug: http://bugzilla.backup-manager.org/show_bug.cgi?id=219
Bug-Debian: http://bugs.debian.org/532976

---
 lib/backup-methods.sh |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

Index: backup-manager/lib/backup-methods.sh
===================================================================
--- backup-manager.orig/lib/backup-methods.sh
+++ backup-manager/lib/backup-methods.sh
@@ -636,7 +636,12 @@ function build_clear_archive
 
         # the common commandline
         *)
-            BM__CURRENT_COMMAND="generic"
+            # tar, tar.gz, tar.bz2, tar.whatever
+            if [[ "${BM_TARBALL_FILETYPE:0:3}" == "tar" ]] ; then
+                BM__CURRENT_COMMAND="tar"
+            else
+                BM__CURRENT_COMMAND="generic"
+            fi
             debug "$command $file_to_create \"$target\" > $logfile 2>&1"
             tail_logfile "$logfile"
             debug "$command $file_to_create \"$target\""
